### ISSUE TITLE
Returned MessageId is not always the last message on sendFile/sendImage

### ISSUE THREAD

**Issue by JoeDozan24** [open-wa/wa-automate-nodejs#362]
**Describe the bug**
Returned MessageId is not always the last message on sendFile/sendImage

**Steps to Reproduce**
1. Start open-wa
2. Try to send file/image
2a. If File size is small (~200KB or less) it returns the messageId for the latest message
2b. If File size is quite large (1MB or more, depending Internet speed too) it returns the second last messageId of a chat

**create() code**
This is the code you use to create the client. e.g
```javascript
await sulla.create({
	sessionId		: session,
	customUserAgent	: uaOverride,
	...puppeteerOverride
});
```

**Expected behavior**
Returns the last messageId

**DEBUG INFO**
This is the info printed to the console when you start your app. It should look like this
```
Debug Info {
  WA_VERSION: 2.2013.7,
  PAGE_UA: WhatsApp/0.4.613 Mozilla/5.0 (Macintosh; Intel Mac OS X  10_15_2) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/79.0.3945.88 Safari/537.36,
  WA_AUTOMATE_VERSION: 1.8.80,
  BROWSER_VERSION: Chrome/80.0.3987.0
}
```

**Screenshots**
![sendFile messageId](https://user-images.githubusercontent.com/58735845/80207756-f2fafb00-8658-11ea-9d06-218df0ad2c4c.png)

**Host (please complete the following information):**
 - OS: Windows

**Additional context**
I add `return` on the sendFile API from this:
```js
Whatsapp.prototype.sendFile = function (to, base64, filename, caption, quotedMsgId) {
        return __awaiter(this, void 0, void 0, function () {
            return __generator(this, function (_a) {
                switch (_a.label) {
                    case 0: return [4, this.page.evaluate(function (_a) {
                            var to = _a.to, base64 = _a.base64, filename = _a.filename, caption = _a.caption, quotedMsgId = _a.quotedMsgId;
                            WAPI.sendImage(base64, to, filename, caption, quotedMsgId);
                        }, { to: to, base64: base64, filename: filename, caption: caption, quotedMsgId: quotedMsgId })];
                    case 1: return [2, _a.sent()];
                }
            });
        });
    };
```
into this:
```js
Whatsapp.prototype.sendFile = function (to, base64, filename, caption, quotedMsgId) {
        return __awaiter(this, void 0, void 0, function () {
            return __generator(this, function (_a) {
                switch (_a.label) {
                    case 0: return [4, this.page.evaluate(function (_a) {
                            var to = _a.to, base64 = _a.base64, filename = _a.filename, caption = _a.caption, quotedMsgId = _a.quotedMsgId;
                            return WAPI.sendImage(base64, to, filename, caption, quotedMsgId);
                        }, { to: to, base64: base64, filename: filename, caption: caption, quotedMsgId: quotedMsgId })];
                    case 1: return [2, _a.sent()];
                }
            });
        });
    };
```
Thought the sendImage on WAPI already returned latest message Id.

**Comment by smashah** [comment]
https://github.com/open-wa/wa-automate-nodejs/issues/298#issuecomment-612371969
 #298

**Comment by smashah** [comment]
@github-actions run

<details>
<summary>⚡ Release!  ⚡</summary>

```js
(async () => {
function exec(cmd) {
  console.log(execSync(cmd).toString());
}

// Config
const gitUserEmail = "github-actions[bot]@users.noreply.github.com";
const gitUserName = "github-actions[bot]";

exec(`echo "//registry.npmjs.org/:_authToken=$NPM_TOKEN" > .npmrc`);
exec(`git config --global user.email "${gitUserEmail}"`);
exec(`git config --global user.name "${gitUserName}"`);
exec(`npm i`);
exec(`npm run release-ci`);

//comment on the issue
var result = execSync(`npx auto-changelog -o ./tempchangelog.txt --commit-limit false --template ./compact-keepachangelog.hbs --stdout`).toString();

    await postComment(result);

//create changelog image
exec(`npm run release-image`);
exec(`git commit -a -m 'updated release-image'`);
exec(`git push --force`);
  })();
```
</details>

**Comment by smashah** [comment]
# Changelog

## 🚀 Release [1.9.1](https://github.com/open-wa/wa-automate-nodejs/compare/1.9.0...1.9.1) (2020-04-25)

- ts fixes #362 [`#362`](https://github.com/open-wa/wa-automate-nodejs/issues/362)
- Release 1.9.1 [`6d08ef8`](https://github.com/open-wa/wa-automate-nodejs/commit/6d08ef8fa127acff2de0c9998235d962f962e88b)
- feature: Optionally wait for id when sending file/image #362 #298 [`8b82448`](https://github.com/open-wa/wa-automate-nodejs/commit/8b82448883931d60b52a7316843ddc57e677e9a9)
- update release image [`4548dbc`](https://github.com/open-wa/wa-automate-nodejs/commit/4548dbc13cc972e831b51c374baa95d6ccc9f281)

**Comment by smashah** [comment]
@JoeDozan24 I stand by the advice shared in #298 however you can now wait for the ID using the 6th parameter:

```javascript
//The fifth param is null to show no quoted message (i.e this is not a reply to another message)
const id = await client.sendImage('447777777777@c.us',;data......','file.jpeg','cool picture',null, true)
```
If after 10 seconds it cannot correctly get the id it will return with `true` (Also, without the 6th param being true, it will not return the id by default).

@blackchineykh
